openedx/openedx-platform
# feat: add django admin page for survey report

### Description:

This PR adds the django model admin to visualize the data from the SurveyReport model (https://github.com/openedx/edx-platform/pull/31183).

ISSUE:
https://github.com/openedx/tcril-engineering/issues/349

APPROVED DESIGN:
https://docs.google.com/document/d/1tm1O5BQsAwekphI5IN9pBn8behQDZigXePLGqCl5he8

### DECISIONS:
- We removed the add and save buttons because the user shouldn't to create or edit the original report data to protect data consistency.

### How to test
With a dev environment running, you should run the command:
```bash
python manage.py lms generate_report
```
and you could check the reports in `/admin/survey_report/surveyreport/`
